test/output-files.js

```diff
@@ -18,8 +19,8 @@ describe('output-files', () => {
 
     // Since block-else-not-covered was generated by the above line, this
     // should make a new file with -1 appended to the name
-    var newPath = outputFiles.rewritePath('./sample_js/block-else-not-covered-1.js')
-    newPath.should.include(storagePath + '/block-else-not-covered-1.js')
+    var newPath = outputFiles.rewritePath('file:////./sample_js/block-else-not-covered-1.js')
```

Comment:
I think we should probably split this into two tests:

1. assumes file type specifier when no prefix is provided.
2. handles `file:///` specifier.